packages/main/src/services/@linux/system-api.ts

```diff
@@ -76,7 +76,8 @@ class LinuxMediaService extends MprisService implements NuclearApi {
       'mpris:trackid': this.objectPath(`track/${index}`),
       'mpris:artUrl': track.thumbnail,
       'xesam:title': track.name,
-      'xesam:artist': [track.artist]
+      // @ts-expect-error NuclearMeta is not a descendant of Track, but Track.artists should be available here anyway
+      'xesam:artist': track.artists ?? [track.artist]
```

Comment:
NuclearMeta should be updated the same way. I'm afraid this is also going to need that:
- Scanned local library tracks will need to be migrated on first run
- Scanner will need to return data in this new format

Reply:
I adjusted the obvious bits, although blind (no manual testing).

As written in the Discord thread and as I have no usage experience with that part of the program, I think it's better you take this part.

Reply:
[Line 203](https://github.com/nukeop/nuclear/blob/129ee02ce85b68d8f15c46e6d3b3c90da4bda1bc/packages/main/src/services/%40linux/system-api.ts#L203) should be updated to `artists`  as well, otherwise it crashes on start.